PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Views/VideoViewController.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  VideoViewController.swift
+//  ControlsLockScreenSampleApp
+//
+//  Created by Carlos Ceja on 10/18/17.
+//  Copyright © 2017 Ooyala. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+import MediaPlayer
+
+class VideoViewController: OOOoyalaPlayerViewController {
+  
+  @IBOutlet weak var playerView: UIView!
+  
+  // properties for the video
+  private var option: PlayerSelectionOption!
+  
+  // properties required for a Fairplay asset
+  private var apiKey: String?
+  private var apiSecret: String?
+  
+  internal static func instantiate(with option: PlayerSelectionOption) -> VideoViewController {
+    let vc: VideoViewController = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "VideoViewController") as! VideoViewController
+    vc.option = option
+    return vc
+  }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.title = option.getTitle()
+    
+    if option.getEmbedTokenGenerator() != nil {
```

Comment:
you can simplify these if statements. Example:
```
if let embedTokenGen = option.getEmbedTokenGenerator() {
  if let embedTokenGen = embedTokenGen as? BasicEmbedTokenGenerator {
    apiKey = embedTokenGen.getApiKey()
    apiSecret = embedTokenGen.getApiSecret()
  } else {
    apiKey = "API_KEY"
    apiSecret = "API_SECRET"
  }
  // ... rest of the code
}
```

Reply:
Done.